base/server/python/pki/server/deployment/pkiconfig.py

```diff
@@ -40,7 +40,6 @@
 PKI_DEPLOYMENT_DEFAULT_USER = "pkiuser"
 
 PKI_SUBSYSTEMS = ["CA", "KRA", "OCSP", "RA", "TKS", "TPS"]
```

Comment:
The RA can be removed from PKI_SUBSYSTEMS since it was already dropped several years ago.